sse2neon.h

```diff
@@ -1744,6 +1749,30 @@ FORCE_INLINE void _mm_free(void *addr)
     free(addr);
 }
 
+// Macro: Get the flush zero bits from the MXCSR control and status register.
+// The flush zero may contain any of the following flags: _MM_FLUSH_ZERO_ON or
+// _MM_FLUSH_ZERO_OFF
+// https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_MM_GET_FLUSH_ZERO_MODE
+FORCE_INLINE unsigned int _MM_GET_FLUSH_ZERO_MODE()
```

Comment:
Both `_MM_GET_FLUSH_ZERO_MODE` and `_MM_SET_FLUSH_ZERO_MODE` were referred to be macros in Intel Intrinsics Guide. Maybe we should follow the behavior. That is, we can implement static inline functions such as `_sse2neon_get_flush_zero_mode` and define the corresponding macros.

Reply:
Done.